### PR TITLE
srikanth changes for report card

### PR DESCRIPTION
**Pull request by srikanth2172**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,14 +24,12 @@
...
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
-			<version>2.0.3</version>
+			<version>2.1.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-
-
```
Comment (Manoj152A) [inline pom.xml:34]: please remove the space

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -12,12 +12,6 @@ spring.datasource.url=jdbc:mysql://localhost:3306/stu
 spring.datasource.username=root
 spring.datasource.password=7842789233@Siva
 
-
-
-
-
```
Comment (Manoj152A) [inline src/main/resources/application.properties:18]: please remove  the space here too

**Review by Manoj152A — APPROVED** [review]

File: src/main/java/com/learn/student/userdetails/Reportcardapi/Model/Student.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+@Entity
+@Table(name="students")
```
Comment (Manoj152A) [inline src/main/java/com/learn/student/userdetails/Reportcardapi/Model/Student.java:14]: Can you please change the table name cause we already have the table name for the student in the same database
and  can i know where the student data is coming from cause i havent seen any queries or table related to the code

**Review by Manoj152A — CHANGES_REQUESTED** [review]

**Review by nikhilmittapally — COMMENTED** [review]
Change tables and use joints for the SQL , then it will retreive the courses from the enrollments

**Pull request closed without merging**